tests/FakerDotNet.Tests/Fakers/RandomFakerTests.cs

```diff
@@ -1,65 +1,65 @@
-using System.Linq;
```

Comment:
Not sure why it says that this file has changed :sweat_smile:

Reply:
I can't seem to revert the non change :/
Might have been a line ending change.